relationships/synthesis/EXT-SERVICE-to-INFRA-ELASTICACHE-REDIS-CLUSTER

```diff
@@ -0,0 +1,54 @@
+relationships:
+  - name: extServiceCallsInfraElastiCacheRedisCluster
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Metric" ]
+      - attribute: db.system
+        anyOf: [ "redis" ]
+    relationship:
+      expires: P75M
+      relationshipType: CALLS
+      source:
+        extractGuid:
+          attribute: entity.guid
+      target:
+        lookupGuid:
+          candidateCategory: AWSELASTICACHEREDISCLUSTER
+          fields:
+            - field: cloud.resource_endpoint_address
+              attribute: net.peer.name
+            - field: cloud.resource_endpoint_port
+              attribute: net.peer.port
+            - field: cloud.elasticache_engine
+              attribute: db.system
+              
+- name: apmServiceCallsInfraElastiCacheRedisCluster
+  version: "1"
+  origins:
+    - APM Metrics
+  conditions:
```

Comment:
Looks like we have data matching this condition:
https://staging.onenr.io/0gR7nyPdpwo
https://staging.onenr.io/0VRVAbEP6wa

Reply:
Please, in your next PR, include these kind of queries in the description. It simplifies a lot the review process.